server/bundler/utils.js

```diff
@@ -9,7 +9,7 @@ function getAssets( stats ) {
 			name: chunk.names[0],
 			hash: chunk.hash,
 			file: filename,
-			url: path.resolve( stats.publicPath, filename )
+			url: stats.publicPath + filename,
```

Comment:
Just a note, in case someone wonders why this change - the old version is incorrect. `path.resolve` should be used for real paths. But here `stats.publicPath` is `/calypso`, a prefix for URLs for assets. This worked on *nix systems because it looks like a Unix path, but on Windows this invocation resulted in a path like `C:\\calypso\\bundle-hash.js` which is definitely not what we want :)